<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QWidget" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>805</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>805</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>805</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#GameWindow {
border-image: url(&quot;:/resources/images/background_hd.png&quot;);
}</string>
  </property>
  <widget class="QLCDNumber" name="steps_lcd">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>782</y>
     <width>31</width>
     <height>18</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>888</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="level_lcd">
   <property name="geometry">
    <rect>
     <x>663</x>
     <y>782</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>88</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="minstep_lcd">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>782</y>
     <width>30</width>
     <height>18</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>888</number>
   </property>
  </widget>
  <widget class="QLabel" name="outlet">
   <property name="geometry">
    <rect>
     <x>583</x>
     <y>547</y>
     <width>112</width>
     <height>226</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(&quot;:/resources/images/outlet.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="done_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
